app/uk/gov/hmrc/exports/models/declaration/submissions/CancellationStatus.scala

```diff
@@ -58,4 +58,21 @@ object CancellationStatus {
   implicit object CancellationStatusWrites extends Writes[CancellationStatus] {
     def writes(status: CancellationStatus): JsValue = JsString(status.toString)
   }
+
+  case class CancellationResult(status: CancellationStatus, conversationId: Option[String])
+
+  object CancellationResult {
+    implicit val cancellationResultFormat: Format[CancellationResult] = new Format[CancellationResult] {
+      def reads(json: JsValue): JsResult[CancellationResult] =
```

Comment:
Out of curiosity (maybe there's something I do not see :slightly_smiling_face:)... why do you need explicit `reads` and `writes` ?